### ISSUE TITLE
Tooltip text not showing up when using GDI

### ISSUE THREAD

**Issue by bayuperdana** [PoE-TradeMacro/POE-TradeMacro#701]
The tool tip when using ctrl+c / Ctrl+d is working fine but when there's no text inside the window.
I can change the background/border to any colors but not with the text, cause i still can't see it.
Here's the screenshot
![screenshot_1](https://user-images.githubusercontent.com/34747351/38397348-79f11ac2-3967-11e8-8831-192331d51269.png)

AHK version: 1.1.26.01
Windows version: Win 7 64bit

**Comment by aRTy42** [comment]
Do you have that problem with all items?

What colours do you have in your GDI configuration?

**Comment by bayuperdana** [comment]
window 000000 with 90 opacity
border FFEA00 with 90 opacity
text 1203FE with 90 opacity

I've tried changing to another color for those 3, only the text that doesn't seem to change or be visible.
other two works fine.

I've tried with some of my equipped item (text not visible)
I've tried with some of my item on stash (text still not visible)
The tooltip border size also changes based on different item so i think the border works fine.
That come to my conclusion, only the text that's not working for me, cause it's not visible.
